test/zepto.html

```diff
@@ -1965,6 +1965,16 @@
         t.assertEqual('[boom]', el.data('invalid'))
       },
 
+      testFormAttrNull: function(t) {
+        var input = $("#attr_val");
+
+          t.assertNull(input.attr("form"), "Form should be null");
+      },
+        testFormAttrNotNull: function(t){
+            var input = $("#attr_val_outside_form");
+
+            t.assertEqual("attr_val_form", input.attr("form"), "Form should be null");
```

Comment:
Why does this test expect that `input.attr("form")` will result in a string `"attr_val_form"`?